loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -367,8 +367,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 		for (var i in data.children) {
 			var child = data.children[i];
-			if (child.id === 'help')
+			if (child.id === 'help' || (child.id === 'cancel' && data.children.length < 3)) {
```

Comment:
whole patch is ggod, but please remove this part (fixed in the core already)

Reply:
will do, thanks!